### PR TITLE
Clean up: make "osu!team" a regular collocation

### PR DESCRIPTION
**Pull request by TicClick**
part of #1698
thanks to everyone who helped with the translations. other occurrences are in outdated articles, so they will eventually be fixed by future translators

### CODE REVIEW

**Comment by TicClick** [comment]
style errors are in the article which will be removed by #4941 so I don't see any point in fixing it

**Review by peppy — DISMISSED** [review]
seems fine

File: wiki/Beatmap/fr.md [hunk 1]
```diff
@@ -109,4 +109,4 @@ La communauté d'osu! organise régulièrement différents concours pour promouv
 
 *Page principale: [Artistes associés](/wiki/Featured_Artists)*
 
-Les artistes associés (Featured Artists) est un programme de l'[osu!team](/wiki/People/The_Team) visant à licencier la musique de différents artistes pour osu!, les rendant ainsi disponible au mapping. La [liste des artistes associés](https://osu.ppy.sh/beatmaps/artists) du site internet présente chaque artiste impliqué et fournit des modèles pré-paramétrés pour le mapping.
+Les artistes associés (Featured Artists) est un programme de l'[osu! team](/wiki/People/The_Team) visant à licencier la musique de différents artistes pour osu!, les rendant ainsi disponible au mapping. La [liste des artistes associés](https://osu.ppy.sh/beatmaps/artists) du site internet présente chaque artiste impliqué et fournit des modèles pré-paramétrés pour le mapping.
```
Comment (Joehuu) [inline wiki/Beatmap/fr.md:112]:
```suggestion
Les artistes associés (Featured Artists) est un programme de l'[osu! team](/wiki/People/The_Team) visant à licencier la musique de différents artistes pour osu!, les rendant ainsi disponible au mapping. La [liste des artistes associés](https://osu.ppy.sh/beatmaps/artists) du site internet présente chaque artiste impliqué et fournit des modèles pré-paramétrés pour le mapping.
```
```suggestion
Les artistes associés (Featured Artists) est un programme de l'[équipe du jeu](/wiki/People/The_Team) visant à licencier la musique de différents artistes pour osu!, les rendant ainsi disponible au mapping. La [liste des artistes associés](https://osu.ppy.sh/beatmaps/artists) du site internet présente chaque artiste impliqué et fournit des modèles pré-paramétrés pour le mapping.
```

File: wiki/Beatmap/id.md [hunk 2]
```diff
@@ -115,4 +115,4 @@ Komunitas osu! secara rutin mengadakan kontes-kontes yang berbeda untuk mempromo
 
 *Artikel utama: [Featured Artist](/wiki/Featured_Artists)*
 
-Featured Artist adalah program komunitas yang dibuat oleh [osu!team](/wiki/People/The_Team) yang berfokus pada pemberian lisensi musik dari berbagai artis untuk osu! dan membuatnya tersedia untuk mapping. Situs web [daftar featured artist](https://osu.ppy.sh/beatmaps/artists) memamerkan setiap artis yang terlibat sekaligus menyediakan template untuk mapping.
+Featured Artist adalah program komunitas yang dibuat oleh [osu! team](/wiki/People/The_Team) yang berfokus pada pemberian lisensi musik dari berbagai artis untuk osu! dan membuatnya tersedia untuk mapping. Situs web [daftar featured artist](https://osu.ppy.sh/beatmaps/artists) memamerkan setiap artis yang terlibat sekaligus menyediakan template untuk mapping.
```
Comment (Joehuu) [inline wiki/Beatmap/id.md:118]:
```suggestion
Featured Artist adalah program komunitas yang dibuat oleh [tim inti osu!](/wiki/People/The_Team) yang berfokus pada pemberian lisensi musik dari berbagai artis untuk osu! dan membuatnya tersedia untuk mapping. Situs web [daftar featured artist](https://osu.ppy.sh/beatmaps/artists) memamerkan setiap artis yang terlibat sekaligus menyediakan template untuk mapping.
```

**Review by Joehuu — CHANGES_REQUESTED** [review]

**Review by Joehuu — APPROVED** [review]

**Pull request merged**